### PR TITLE
Updates SNS_Topic Name

### PR DESCRIPTION
**Pull request by Benbentwo**
## what
* Updates sns_topic name
  From `CloudTrailBreach-threshold-alerts` to `${namespace}-${env}-${stage}-cloudwatch-cloudtrail-breach`

## why
* follow naming convention

### CODE REVIEW

**Review by osterman — APPROVED** [review]

File: main.tf [hunk 1]
```diff
@@ -56,9 +56,18 @@ data "aws_iam_policy_document" "sns_kms_key_policy" {
   }
 }
 
+module "aws_sns_topic_label" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+  count   = local.enabled ? 1 : 0
```
Comment (nitrocode) [inline main.tf:62]: Isn't this already covered by the `context` ?

Comment (nitrocode) [reply main.tf:62]: Also I do not believe `local.enabled` is defined.

Comment (Benbentwo) [reply main.tf]: it was, but removed it anyways

**Review by nitrocode — APPROVED** [review]

**Pull request merged**